From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-userfaultfd-support-wp-on-multiple-vmas-fix
Date: Thu Feb 23 03:13:06 PM PST 2023

s/VM_WARN_ON_ONCE/VM_WARN_ONCE/ to fix build

Cc: Muhammad Usama Anjum <usama.anjum@collabora.com>
Cc: Peter Xu <peterx@redhat.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Paul Gofman <pgofman@codeweavers.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/userfaultfd.c~mm-userfaultfd-support-wp-on-multiple-vmas-fix
+++ a/mm/userfaultfd.c
@@ -717,7 +717,7 @@ long uffd_wp_range(struct mm_struct *dst
 	struct mmu_gather tlb;
 	long ret;
 
-	VM_WARN_ON_ONCE(start < dst_vma->vm_start || start + len > dst_vma->vm_end,
+	VM_WARN_ONCE(start < dst_vma->vm_start || start + len > dst_vma->vm_end,
 			"The address range exceeds VMA boundary.\n");
 	if (enable_wp)
 		mm_cp_flags = MM_CP_UFFD_WP;
_
